.github/workflows/root-ci-config/buildconfig/ubuntu2404-cuda.txt

```diff
@@ -0,0 +1,19 @@
+CMAKE_BUILD_TYPE=RelWithDebInfo
+pythia8=OFF
+cuda=ON
+ccache=ON
+cudnn=ON
+tmva-cpu=ON
+tmva-gpu=ON
+tmva-pymva=ON
+tmva=ON
+fortran=OFF
+gdml=OFF
+mysql=OFF
+spectrum=OFF
+sqlite=OFF
+test_distrdf_dask=OFF
+test_distrdf_pyspark=OFF
+unfold=OFF
+webgui=OFF
+xml=OFF
```

Comment:
`stressHistFactory` requires xml, hence the test failure. Thanks for uncovering that!

Reply:
does stressHistFactory use GPUs? If not, do you think it can in the medium term? If not we can think about disabling the test if xml is off perhaps.

Reply:
It doesn't use GPUs, and it also would make no sense to do this. Its a binned fit with 2 bins.

Actually, we already test these kind of models with the GPU in `stressRooStats`! So also from that perspective it's not worth it to run `stressHistFactory` on the GPU.